training-maquetado/index.html

```diff
@@ -8,6 +8,14 @@
   <link rel="stylesheet" type="text/css" href="application.css">
 </head>
 <body>
+  <header class="navbar d-flex align-center justify-between">
+    <div class="navbar-logo">
```

Comment:
I think you can remove this div and add this class directly to the child element